### ISSUE TITLE
Multi-language Wrappers

### ISSUE THREAD

**Issue by axsaucedo** [SeldonIO/MLServer#183]
Create multi-language wrappers that can run Java, C++ and R models. For this, we can leverage the existing research in Seldon Core which leverages tools like JNI and PyBind to bridge from Python to other runtimes.

**Comment by indranilr** [comment]
Adding to the above issue as indicated in [https://github.com/SeldonIO/seldon-core/issues/4531](url)

[MLeap](https://github.com/combust/mleap) is used to deploy Spark mllib models (including Spark Xgboost and others) for inference through Java runtime. With Seldon core the Java language wrapper could be used to wrap MLeap format models and deploy using Java JNI wrapper. The JNI wrapper however had many vulnerabilities https://github.com/SeldonIO/seldon-core/issues/4103.

Please support deployment of MLeap serialized Spark MLlib models in MLServer so that Java JNI wrapper vulnerabilities can be avoided.

**Comment by adriangonz** [comment]
Hey @indranilr ,

Thanks for expressing interest on this one! 

Just as a head up, this is on the roadmap, but probably won't come through until the `1.4.0` release.

**Comment by dtpryce** [comment]
Just out of interest would this include Julia support?

**Comment by adriangonz** [comment]
We've already done something along these lines for C++ and Java for SCv1 so that would be the initial priority. We are not planning to explore adding support for Julia yet.

**Comment by dtpryce** [comment]
Totally understood! Will be good to see those first few languages and we could always try to mimic for Julia if we decide we really need it :) thank @adriangonz!

**Comment by indranilr** [comment]
> Hey @indranilr ,
> 
> Thanks for expressing interest on this one!
> 
> Just as a head up, this is on the roadmap, but probably won't come through until the `1.4.0` release.

@adriangonz since 1.4.0 is released now, has the support for MLeap serialized Spark Xgboost model incorporated ? Could you please point to relevant documentation ?

**Comment by ramonpzg** [comment]
Hi @indranilr -- We recently moved things around in the roadmap of MLServer and have started dedicating more capacity to improving different features of it and its runtime. It might take us a couple more versions before we can fully dedicate time for multi-language wrappers (it would definitely be very valuable to add these :raised_hands: ).
